pyproject.toml

```diff
@@ -43,6 +43,7 @@ PyYAML = "*"
 [tool.poetry.dev-dependencies]
 pytest = "*"
 pytest-cov = "*"
+pytest-error-for-skips = "*"
```

Comment:
Do we really need an extra dependency for this? What was wrong with the `skipif` approach?

Reply:
`skipif` skips tests. This plugin lets pytest fail if anything was skipped. Basically to make sure that for at least one Python version, no test is skipped. Unfortunately, pytest does not support that out of the box...